### PR TITLE
Added samples to Merge documents without import headers,Resize list character, Insert author name

### PR DESCRIPTION
**Pull request by elizabethatienoo**
Added KB samples. The samples is:
Merge-documents-without-import-headers,Insert-author-name,Resize-list-character

### CODE REVIEW

File: Paragraphs/Resize-list-character/.NET-Standard/Resize-list-character/Program.cs [hunk 1]
```diff
@@ -35,5 +30,67 @@ static void Main(string[] args)
                 }
             }
         }
+        /// <summary>
+        /// Iterate through source document section.
```
Comment (Suriya-Balamurugan) [inline Paragraphs/Resize-list-character/.NET-Standard/Resize-list-character/Program.cs:34]: Iterate through the headers and footers.

File: Paragraphs/Resize-list-character/.NET-Standard/Resize-list-character/Program.cs [hunk 2]
```diff
@@ -13,18 +13,13 @@ static void Main(string[] args)
                 //Open the template document.
                 using (WordDocument document = new WordDocument(fileStreamPath, FormatType.Automatic))
                 {
-                    //Get the textbody content and adds it to document section.
-                    WTextBody textbody = document.Sections[0].Body;
-                    //Iterate through thedocument paragraphs.
-                    foreach (WParagraph paragraph in textbody.Paragraphs)
+                    foreach (WSection section in document.Sections)
```
Comment (Suriya-Balamurugan) [inline Paragraphs/Resize-list-character/.NET-Standard/Resize-list-character/Program.cs:16]: Iterate through the each section of the Word document.

File: Paragraphs/Resize-list-character/.NET-Standard/Resize-list-character/Program.cs [hunk 3]
```diff
@@ -13,18 +13,13 @@ static void Main(string[] args)
...
                     {
-                        //Get the symbol from the paragraph items.
-                        foreach (ParagraphItem item in paragraph.ChildEntities)
-                        {
-                            //Change the list character size.
-                            if (paragraph.ListFormat != null && paragraph.ListFormat.CurrentListLevel != null)
-                                paragraph.ListFormat.CurrentListLevel.CharacterFormat.FontSize = 25;
-                        }
+                        //Accesse the Body of section where all the contents in document are apart.
```
Comment (Suriya-Balamurugan) [inline Paragraphs/Resize-list-character/.NET-Standard/Resize-list-character/Program.cs:18]: Access

File: Paragraphs/Resize-list-character/.NET-Standard/Resize-list-character/Program.cs [hunk 4]
```diff
@@ -13,18 +13,13 @@ static void Main(string[] args)
...
+                        WTextBody sectionBody = section.Body;
+                        IterateTextBody(sectionBody);
+                        IterateHeaderFooter(section);
```
Comment (Suriya-Balamurugan) [inline Paragraphs/Resize-list-character/.NET-Standard/Resize-list-character/Program.cs:21]: //Iterate through the headers and footers

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by Suriya-Balamurugan — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**